src/factorio/Main.java

```diff
@@ -10,11 +10,11 @@ public class Main extends Application {
 
     @Override
     public void start(Stage primaryStage) throws Exception {
-        primaryStage.setScene(new Scene(FXMLLoader.load(getClass().getResource("gui/BitGIFGenerator.fxml")), 764, 541));
+        primaryStage.setScene(new Scene(FXMLLoader.load(getClass().getResource("gui/BitGIFGenerator.fxml")), 780-16, 580-39));
```

Comment:
It would be better to either use the true values instead of an arrhythmic expression,
or explain on why these offsets are needed for these specific ones.

Reply:
We sure can use the true values. It's just that the window size in the end is offset by this much.
So I thought it would be easier to edit the actual size than calculating it, because I keep forgetting this offset.
(I could annotate it)

Reply:
Ok thanks, if it is an offset it is worth placing a comment next to it; so it is not just magic numbers.